actors/controllers.py

```diff
@@ -113,7 +117,12 @@ def cron_execution_datetime(self, actor):
         logger.debug(f"cron execution is {cron_execution}")
         # Return true/false comparing now with the next cron execution
         logger.debug(f"does cron == now? {cron_execution == now}")
-        return cron_execution == now
+        if cron_execution == now:
```

Comment:
If you were to keep this style of logic, it would be appreciated to use something more verbose than ints, readability is harder like this.